src/store/apartments/mutations.ts

```diff
@@ -1,10 +1,25 @@
 import { MutationTree } from 'vuex';
 import { ApartmentsStateInterface, ApartmentsMutationsEnum, Mutations } from '@/store/apartments/types';
+import { ApartmentModel } from './models/apartment.model';
+import { OwnerModel } from '@/shared/models/owner.model';
 
 export const mutations: MutationTree<ApartmentsStateInterface> & Mutations = {
   [ApartmentsMutationsEnum.SET_APARTMENTS]: (state, payload) => {
     state.apartments = payload;
   },
+  [ApartmentsMutationsEnum.GET_APARTMENT_BY_ID]: (state, payload) => {
```

Comment:
So the mutation doesn't get anything, but it sets apartmentInfo
Maybe it should be called SET_APARTMENT_INFO?